DXgovernance/davi-monorepo
# Feat: useVoterLockTimestamp and useProposalVotesOfVoter hooks

## Description

Extracted `useVoterLockTimestamp` and `useProposalVotesOfVoter` hooks to store. Also added typings and tests.

Fixes #52 , #53 

## Type of change

- [ ] Documentation (This change updates documentation)
- [ ] Release (This pull request refers to a new version release)
- [ ] Other (This change updates documentation)

## How Has This Been Tested?

Added unit tests for `useProposalVotesOfVoter`, and updated tests for `useVoterLockTimestamp`.

This changes should not impact the previous functionality, so everything must work the same.

To test `useProposalVotesOfVoter`
 - Go to a proposal and vote. It should show you the option and percentage you voted with
 - In the "all proposals" view, it should show you the tick that indicates that you voted on that proposal

To test `useVoterLockTimestamp` we need a guild with ERC20 locking functionality (I think there isn't one deployed):
 - Should show you the time your tockens will be unlocked
 - In a REP guild it won't show. This is handled in the UI, so I don't think we can test this

## Checklist:
